psychopy/parallel/__init__.py

```diff
@@ -37,16 +37,24 @@
     ParallelPort = PParallelLinux
 elif sys.platform == 'win32':
     from ctypes import windll
-    if hasattr(windll, 'inpout32'):
+    try:
+        hasattr(windll, 'inpout32')
         from ._inpout32 import PParallelInpOut32
         ParallelPort = PParallelInpOut32
-    elif hasattr(windll, 'dlportio'):
-        from ._dlportio import PParallelDLPortIO
-        ParallelPort = PParallelDLPortIO
-    else:
-        logging.warning("psychopy.parallel has been imported but no "
-                        "parallel port driver found. Install either "
-                        "inpout32 or dlportio")
+    except OSError:
+        try:
+            hasattr(windll, 'inpoutx64')
+            from ._inpoutx64 import PParallelInpOutx64
+            ParallelPort = PParallelInpOutx64
+        except OSError:
+            try:
+                hasattr(windll, 'dlportio')
+                from ._dlportio import PParallelDLPortIO
+                ParallelPort = PParallelDLPortIO
+            except OSError:
```

Comment:
This is super convoluted now. If it stays that way, we should at least add a comment above all those try/except blocks to explain what's going on.